.github/workflows/gui-checks.yml

```diff
@@ -89,6 +89,13 @@ jobs:
           fail-on-error: false
           fail-on-warning: false
 
+      - name: 📥 Download Enso Engine
+        continue-on-error: true
+        working-directory: app/project-manager-shim
+        run: pnpm run download-engine
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Is this a check if we're able to download engine, or we need to download engine for one of these checks?

Reply:
We need the enso executable to run the `ProjectService.test.ts` test suite